pre_commit/languages/python.py

```diff
@@ -137,7 +137,7 @@ def norm_version(version: str) -> str | None:
     elif _sys_executable_matches(version):  # virtualenv defaults to our exe
         return None
 
-    if os.name == 'nt':  # pragma: no cover (windows)
+    if sys.platform == 'win32':  # pragma: no cover (windows)
```

Comment:
this can be `# pragma: win32 cover`

Reply:
It borked the tests, so I changed it back :D. I guess this conversation still counts as "resolved"?